docs/project/contribute.md

```diff
@@ -64,13 +64,56 @@ Search is by algolia, the configuration is at <https://github.com/algolia/docsea
 
 ## Running the site locally:
 
-All Contributors uses Docusaurus to maintain our documentation website. For more info visit the [Docusaurus Website](https://docusaurus.io)
+All Contributors currently uses [Docusaurus 1.x](https://docusaurus.io) for our documentation website.
 
-To get started locally:
+To build the docs locally:
 
-- `yarn install`
-- `yarn start`
-- go to `http://localhost:3000` (Refresh the page for your changes to be reflected)
+* <https://tutorial.docusaurus.io/docs/intro>
+
+1. Make sure `npm` is installed on your machine. Use the [nodejs documentation](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) if you aren't sure how to install `npm`.
+
+To check that npm is installed run the following in your favorite shell:
+
+`npm -v`
+
+Next, make sure you have atleast node version 0.18 or higher. To check the version,
```

Comment:
```suggestion
Next, make sure you have at least node version 0.22 (LTS) or higher. To check the version,
```

I would recommend the current LTS version. 0.18 is no longer maintained and is EOL.

Reply:
Good suggestion! I am building locally using node 22. I'm VERY WORRIED about upgrading too much else until we bump to a newer version of Docusaurus!!